templates/quickstart-bitbucket-dc-with-vpc.template.yaml

```diff
@@ -163,7 +163,7 @@ Parameters:
       It cannot start or end with a hyphen (-).
     Type: String
   QSS3KeyPrefix:
-    Default: ''
+    Default: 'quickstart-atlassian-bitbucket'
```

Comment:
QSS3KeyPrefix is expected to end with a `/`
See example here:  `${QSS3KeyPrefix}templates`
https://github.com/aws-quickstart/quickstart-atlassian-bitbucket/blob/develop/templates/quickstart-bitbucket-dc-with-vpc.template.yaml#L484